### DISCUSSION TITLE
Lesson 11: ValueError: Gas estimation failed

### DISCUSSION THREAD

**Discussion by peerreez** [smartcontractkit/full-blockchain-solidity-course-py#1897] [Q&A]
ValueError: Gas estimation failed: 'Unknown typed error: 0x8129bbcd'. This transaction will likely revert. If you wish to broadcast, you must set the gas limit manually.
I am trying to run the file deploy_and_create.py. Out of desperation I have the same code as in the repository.
For this to work you need LINK or WETH token.
![errorestimationfailed](https://github.com/smartcontractkit/full-blockchain-solidity-course-py/assets/92093034/434fb957-f9d0-4110-bc1b-01b025a310eb)

Here is my repository:
https://github.com/peerreez/brownie-nft-demo-v1-course

Sorry for the ignorance

**Comment by Frael98** [comment]
I'm stuck in the same error 😥, please if you resolved it could you share the solution please, thank you
